_monitoring-plugins/ad/settings.md

```diff
@@ -45,3 +45,5 @@ plugins.anomaly_detection.delete_anomaly_result_when_delete_detector | False | W
 plugins.anomaly_detection.dedicated_cache_size | 10 | If the real-time analysis of a high cardinality detector starts successfully, the anomaly detection plugin guarantees keeping 10 (dynamically adjustable via this setting) entities' models in memory per node. If the number of entities exceeds this limit, the plugin puts the extra entities' models in a memory space shared by all detectors. The actual number of entities varies based on the memory that you've available and the frequencies of the entities. If you'd like the plugin to guarantee keeping more entities' models in memory and if you're cluster has sufficient memory, you can increase this setting value.
 plugins.anomaly_detection.max_concurrent_preview | 2 | The maximum number of concurrent previews. You can use this setting to limit resource usage.
 plugins.anomaly_detection.model_max_size_percent | 0.1 | The upper bound of the memory percentage for a model.
+plugins.anomaly_detection.door_keeper_in_cache.enabled | False | We have a bloom filter placed in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. This setting allows you to either enable the bloom filter or not.
```

Comment:
Can we say "The anomaly detection plugin has a bloom filter"? To avoid the use of "we".
```suggestion
plugins.anomaly_detection.door_keeper_in_cache.enabled | False | The anomaly detection plugin has a bloom filter placed in front of an inactive entity cache to filter out unpopular items that are not likely to appear more than once. This setting allows you to either enable or disable the bloom filter.
```